src/main/java/org/galatea/starter/domain/FinnhubQuoteEntity.java

```diff
@@ -0,0 +1,29 @@
+package org.galatea.starter.domain;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.time.Instant;
+
+@Entity
+@Table(name = "finnhub_quote_cache")
```

Comment:
Bit of a nit pick but I would avoid calling a db table a cache. A cache to me implies that it's associated w/ in memory storage.